### PR TITLE
Up List show your created Ups and accepted Ups

### PR DESCRIPTION
**Pull request by pingzh**

### CODE REVIEW

File: Groupon UP/Groupon UP/AppDelegate.swift [hunk 1]
```diff
@@ -20,7 +20,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window = UIWindow(frame: UIScreen.mainScreen().bounds)
         window!.backgroundColor = UPBackgroundColor
         window!.tintColor = UPTintColor
-        window!.rootViewController = InitialViewController()
+        window!.rootViewController = InitialViewController()//UINavigationController(rootViewController: UPListViewController())
```
Comment (buy) [inline Groupon UP/Groupon UP/AppDelegate.swift]: remove the comment?

File: Groupon UP/Groupon UP/UPListViewController.swift [hunk 2]
```diff
@@ -7,29 +7,203 @@
...
     }
-
+    
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        initData()
+    }
+    
+    func addSubViews() {
+        navigationItem.title = "Your Ups"
```
Comment (buy) [inline Groupon UP/Groupon UP/UPListViewController.swift]: maybe we should use `My Groupon UP`

**Comment by buy** [comment]
other than the comments above - LGTM

File: Groupon UP/Groupon UP/UPListViewController.swift (hunk 2 above)
Comment (pingzh) [inline Groupon UP/Groupon UP/UPListViewController.swift]: good call

File: Groupon UP/Groupon UP/AppDelegate.swift (hunk 1 above)
Comment (pingzh) [inline Groupon UP/Groupon UP/AppDelegate.swift]: Yep

**Pull request merged**